ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -233,27 +232,99 @@ const LocalStandardDisks = ({ idPrefix, setIsFormValid, onAddErrorNotification }
         }
     ));
 
+    const setSelectionForAllDisks = (value) => {
+        setDisks(Object.keys(disks).reduce((acc, cur) => ({ ...acc, [cur]: value }), {}));
+    };
+
+    const DropdownSplitButtonText = () => {
+        const [isOpen, setIsOpen] = React.useState(false);
+
+        const onToggle = (isOpen) => {
+            setIsOpen(isOpen);
+        };
+
+        const onFocus = () => {
+            const element = document.getElementById("local-disks-bulk-select-toggle");
+            element.focus();
+        };
+
+        const onSelect = () => {
+            setIsOpen(false);
+            onFocus();
+        };
+
+        const dropdownItems = [
+            <DropdownItem
+              key="select-none"
+              component="button"
+              aria-label={_("Select none disk")}
+              onClick={() => setSelectionForAllDisks(false)}
+              id="local-disks-bulk-select-none"
+            >
+                {_("Select none")}
+            </DropdownItem>,
+            <DropdownItem
+              key="select-all"
+              component="button"
+              aria-label={_("Select all disks")}
+              onClick={() => setSelectionForAllDisks(true)}
+              id="local-disks-bulk-select-all"
+            >
+                {_("Select all")}
+            </DropdownItem>,
+        ];
+
+        return (
+            <Dropdown
+              onSelect={onSelect}
+              toggle={
+                  <DropdownToggle
+                    splitButtonItems={[
+                        <DropdownToggleCheckbox
+                          key="select-multiple-split-checkbox"
+                          aria-label={_("Select multiple disks")}
+                          isChecked={selectedDisksCnt > 0}
+                        >
+                            {selectedDisksCnt > 0 ? cockpit.format(_("$0 selected"), selectedDisksCnt) : ""}
```

Comment:
This needs a plural translation. I think the following would work:
```
{selectedDisksCnt > 0 ? cockpit.format(cockpit.ngettext("$0 selected", "$0 selected", selectedDisksCnt), selectedDisksCnt) : ""}
```

In other words, replace `_("$0 selected")` with `cockpit.ngettext("$0 selected", "$0 selected", selectedDisksCnt)`

See also: https://cockpit-project.org/guide/latest/cockpit-locale.html

Reply:
Yes, thank you, silly me was somehow stuck in english only here.

Reply:
Fixed